<launch>

<arg name="sr_source" default="SR_FILE"/>
<arg name="sr_data_file_dir"  default="/home/davidz/work/data/SwissRanger4000/exp/dataset_82"/> 
<arg name="sr_new_file_version" default="false"/>
<arg name="sr_data_prefix"    default="d1"/>
<arg name="sr_data_suffix"    default="bdat"/>
<arg name="sr_start_frame"    default="1"/>
<arg name="sr_end_frame"      default="2200"/>
<arg name="sr_publish_max"    default="5000"/>
<arg name="sr_publish_rate"   default="15"/>
<arg name="sr_syn_from_receiver" default="true"/>
<arg name="sr_cam_it"         default="80"/>


<node pkg="sr_slam" type="sr_publisher" name="sr_publisher" output="screen">
  <param name="sr_source"          value="$(arg sr_source)"/> <!--value="SR_FILE", "SR_CAM" "SR_TCP"/-->

  <param name="sr_data_file_dir"    value="$(arg sr_data_file_dir)"/>
  <param name="sr_start_frame"      value="$(arg sr_start_frame)"/>
  <param name="sr_end_frame"        value="$(arg sr_end_frame)"/>
  
  <param name="sr_new_file_version"   value="$(arg sr_new_file_version)"/>
  <param name="sr_data_prefix"        value="$(arg sr_data_prefix)"/>
  <param name="sr_data_suffix"        value="$(arg sr_data_suffix)"/>
  
  <param name="sr_publish_max"        value="$(arg sr_publish_max)"/>
  <param name="sr_publish_rate"       value="$(arg sr_publish_rate)"/>
  <param name="sr_syn_from_receiver"  value="$(arg sr_syn_from_receiver)"/>

  <param name="sr_publish_intensity"  value="false"/>
  <param name="sr_publish_depth"      value="false"/>
  <param name="sr_display_histogram"  value="false"/>
  <param name="sr_threshold_display"  value="false"/>
  <param name="sr_threshold_value"    value="25"/>

  <param name="sr_cam_ip"                     value="192.168.0.11"/>
  <param name="sr_cam_it"                     value="$(arg sr_cam_it)"/>
</node>


</launch>
